onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -15,6 +15,12 @@ namespace onnxruntime {
 namespace QDQ {
 namespace {
 
+#if defined(_MSC_VER)
+#define FORCEINLINE __forceinline
```

Comment:
it's probably worth making this a commonly available macro.

e.g., maybe move something like this duplicate:

https://github.com/microsoft/onnxruntime/blob/beb2496748b112ba0b2525c14f1093acbd98c7aa/onnxruntime/core/util/matrix_layout.h#L20-L24

to include/onnxruntime/core/common/common.h